.github/workflows/testadapters.yaml

```diff
@@ -108,14 +108,14 @@ jobs:
       - name: Check Meshery pod logs
         run: |
           SECONDS=0
-          end=$((SECONDS+150))
+          end=$((SECONDS+300))
           while [ $SECONDS -lt $end ]; do
-              status=$(kubectl get pods -n meshery | sed -n 2p | tr -s ' ' | cut -d " " -f 3)
+              status=$(kubectl get pods -o wide -n meshery | grep meshery | sed -n 1p | tr -s ' ' | cut -d " " -f 3)
```

Comment:
I have a hard time understanding why `mesheryctl system status` isn't used here. Why isn't it? If something needs changed, are have those changes been requested? // @meshery/mesheryctl-maintainers

Reply:
To get the pod logs, I need the name of the pods. mesheryctl doesn't quite give the name of the pods

Reply:
Please file an enhancement request for a `-v` verbose flag for this command.

Reply:
Good to merge after this issue is filed.

Reply:
https://github.com/meshery/meshery/pull/4845 merge this

Reply:
You don't need the pod-name to get logs, just the component-name is enough.
```
❯ mesheryctl system logs meshery-app-mesh
Starting Meshery logging...
meshery-app-mesh-d8bb6fb98-4qflj        |       Could not load definitions and schemas for static component registeration: lstat /templates/oam/workloads: no such file or directorytime="2022-01-02T08:38:36Z" level=info msg="Adaptor Listening at port: 10005" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:36Z" level=info msg="Starting static component registration..." app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:36Z" level=info msg="Static Component registration completed" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:37Z" level=info msg="Registering latest workload components for version v1.4.2" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       Looking for kubeopenapi-jsonschema in  /root/.meshery/bin
meshery-app-mesh-d8bb6fb98-4qflj        |       Downloading kubeopenapi-jsonschema at /root/.meshery/bin/kubeopenapi-jsonschema...
meshery-app-mesh-d8bb6fb98-4qflj        |       Download Completed
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:39:06Z" level=info msg="Latest workload components successfully registered." app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |
```